components/gcp/dataproc/submit_hadoop_job/README.md

```diff
@@ -72,7 +72,7 @@ To continue to run the sample, make sure that the service account of the noteboo
 !gsutil rm $OUTPUT_GCS_PATH/**
 ```
 
-    CommandException: No URLs matched: gs://hongyes-ml-tests/dataproc/hadoop/output/**
+    CommandException: No URLs matched: gs://hongyes-ml-tests/dataproc/hadoop/output/**
```

Comment:
Nit: Could we change the bucket name from hongyes-ml-tests to ml-pipeline-tests or something?

Reply:
+1

Reply:
Those output cells are mistakenly added. I have removed them.